### ISSUE TITLE
@since tags in the Twisted codebase should meet the Twisted coding standard

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7768]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#7768|
|Type|enhancement|
|Created|2015-02-03 07:03:31Z|
|Branch|https://github.com/twisted/twisted/tree/since-tags-7768|

There were some instances of 15.0.0 in the codebase -- these should be corrected to be 15.0, as per the Twisted Coding Standard in [#7765](https://github.com/twisted/twisted/issues/7765).

<details><summary>Searchable metadata</summary>

```
trac-id__7768 7768
type__enhancement enhancement
reporter__hawkowl hawkowl
priority__normal normal
milestone__None None
branch__branches_since_tags_7768 branches/since-tags-7768
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1422947011427331 1422947011427331
changetime__1445217222500211 1445217222500211
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_1"></a>|@hawkowl commented|
|-|-|

(In [43827]) Branching to since-tags-7768.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

Fixed, putting up for review.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_3"></a>|@glyph set owner to @hawkowl|
|-|-|

Looks good, land please.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl set status to `closed`|
|-|-|

(In [43836]) Merge since-tags-7768: @since tags in the Twisted codebase should meet the Twisted coding standard

Author: hawkowl
Reviewer: glyph
Fixes: [#7768](https://github.com/twisted/twisted/issues/7768)

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl commented|
|-|-|

Ticket retargeted after milestone deleted
